source/clear-linux/tutorials/kernel-boot-params.rst

```diff
@@ -0,0 +1,45 @@
+.. _kernel_boot_params:
+
+Modify kernel boot parameters
+#############################
+
+Changing kernel boot paramters can drastically change the way the OS behaves,
+so be cautious when doing so. |CL| provides a simple way to do this that is
+consistent with its :ref:`stateless` nature. All configuration changes belong
+in the :file:`/etc/kernel/` path. The :command:`clr-boot-manager` generates
+boot entries, so we'll use it to run an update each time we change a boot
+paramter. 
+
+.. warning::
+
+   Be careful when editing boot parameters as the system can be made
+   non-bootable.
+
```

Comment:
I'd suggest `clr-boot-manager get-timeout` and `set-timeout` to set a non-zero boot timeout before experimenting with changes. That way, if boot fails, they can try again, hit `e` at the boot menu to edit the command line, and delete the entries they added.